expect_test.go

```diff
@@ -168,7 +165,15 @@ func TestExpect(t *testing.T) {
 	}
 	// close the pts so we can expect EOF
 	testCloser(t, c.Tty())
-	wg.Wait()
+	waitTestEnd(t, done)
+}
+
+func waitTestEnd(t *testing.T, done <-chan struct{}) {
+	select {
+	case <-done:
+	case <-time.After(100 * time.Millisecond):
```

Comment:
I think this should be at least a few seconds to avoid flakiness on a heavily loaded system.